### PR TITLE
new package: py-pymeeus

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pymeeus/package.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    depends_on('py-more-itertools@4.3.0:', type=('build', 'run'))
+    depends_on('py-pathlib2@2.3.2:', type=('build', 'run'))
+    depends_on('py-pluggy@0.8.0:', type=('build', 'run'))
+    depends_on('py-py@1.7.0:', type=('build', 'run'))
+    depends_on('py-pycodestyle@2.4.0:', type=('build', 'run'))
+    depends_on('py-pyflakes@2.0.0:', type=('build', 'run'))
+    depends_on('py-pytest@4.0.1:', type=('build', 'run'))
+    depends_on('py-pytest-cov@2.6.0:', type=('build', 'run'))
+    depends_on('py-scandir@1.9.0:', type=('build', 'run'))
+    depends_on('py-six@1.11.0:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-pymeeus/package.py]: Where do these dependencies come from?

Comment (healther) [reply var/spack/repos/builtin/packages/py-pymeeus/package.py]: https://github.com/architest/pymeeus/blob/master/requirements.txt

Comment (healther) [reply var/spack/repos/builtin/packages/py-pymeeus/package.py]: that being said: I don't see any references to any of these packages in the imports in the actual code...

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-pymeeus/package.py]: Never trust the `requirements.txt`, always trust the `install_requires` in `setup.py` and `setup.cfg`

Comment (healther) [reply var/spack/repos/builtin/packages/py-pymeeus/package.py]: Okay will drop them

**Pull request merged**